src/js/core/getArrayOfElements.js

```diff
@@ -1,13 +1,17 @@
 /**
 * Returns an array of elements based on the selector input
-* @param {String|Element|Element[]} selector
+* @param {String|Element|Element[]|Object} selector
 * @return {Element[]}
 */
 export default function getArrayOfElements(selector) {
   if (selector instanceof Element) {
     return [selector]
   }
 
+  if (selector instanceof Object) {
```

Comment:
You need to also check `selector.constructor === Object` if you want to check for plain objects, because `Element` inherits from `Object`.